### ISSUE TITLE
Streaming mp3 file 

### ISSUE THREAD

**Issue by math65** [BearWare/TeamTalk5#472]
I cannot broadcast mp3 files. Teamtalk play a sound, and the screen reader says "video of Mathieu MARTIN".

**Comment by bear101** [comment]
Do you use Windows 7?

**Comment by bear101** [comment]
Probably due to Windows 7.
